src/map/anim/Map.FlyTo.js

```diff
@@ -24,7 +24,7 @@ L.Map.include({
 		    rho2 = rho * rho;
 
 		function r(i) {
-			var b = (w1 * w1 - w0 * w0 + (i ? -1 : 1) * rho2 * rho2 * u1 * u1) / (2 * (i ? w1 : w0) * rho2 * u1);
+			var b = (w1 * w1 - w0 * w0 + (i ? -1 : 1) * rho2 * rho2 * u1) / (2 * (i ? w1 : w0) * rho2);
```

Comment:
what's going on here? it can't be removed like that because of the left part of the formula